cpp/ps0001/cpp_ps0001_soln.cpp

```diff
@@ -0,0 +1,29 @@
+#include <iostream>
+using namespace std;
+int main()
+{
+ // promptiung user for 3 numbers to add 1/2
+ unsigned short num1;
+ unsigned short num2;
+ unsigned short num3;
+ unsigned short res;
+
+ cout << "please enter the first number\n";
+ cin >> num1;
+ 
+ cout << "please enter the second number\n";
+ cin >> num2; 
+ 
+ cout << "please enter the third number\n";
+ cin >> num3;
+
+ // outputting the result of the summation process 3/3
+ //processing
+ res = num1 + num2 + num3;
+
+ // output
+ cout << res << "\n";
+ 
+ return 0;
+}
+// great work 😎 8/10 ⭐

```

Comment:
- [ ] program is in the correct path
- [x] no syntax errors
- [x] no bugs occurs while execution
- [x] formatted code to be clear and readable
- [x] commented some phrases for blocks of code